docs/manual/common/guide/production/LightbendPlatform.md

```diff
@@ -10,7 +10,6 @@ Lagom is based on Akka, a set of open-source libraries for designing scalable, r
 
 ### Akka Resilience Enhancements
 
-* [Split Brain Resolver](https://doc.akka.io/docs/akka-enhancements/current/split-brain-resolver.html) - Improved resilience for Akka Cluster applications.
 * [Kubernetes Lease](https://doc.akka.io/docs/akka-enhancements/current/kubernetes-lease.html) - Kubernetes enhanced failure-recovery for Lagom and Akka Cluster apps.
```

Comment:
Kubernetes Lease is part of Akka Management now. Am I right?
https://doc.akka.io/docs/akka-management/current/kubernetes-lease.html

Reply:
It is indeed!